mempool/src/shared_mempool/coordinator.rs

```diff
@@ -218,6 +224,7 @@ fn handle_commit_notification<TransactionValidator>(
     mempool_validator: &Arc<RwLock<TransactionValidator>>,
     use_case_history: &Arc<Mutex<UseCaseHistory>>,
     msg: MempoolCommitNotification,
+    num_committed_txns_recieved_since_peers_updated: &Arc<RwLock<u64>>,
```

Comment:
I think an AtomicU64 should be more efficient. Although it looks like we already use quite a bit of RwLock and Mutex on this code path...

Reply:
Done.